resources/lang/en/Template.properties

```diff
@@ -123,7 +123,7 @@ contactAcceptPrivacyPolicy = "I hereby confirm that I have read the :policy."
 contactCheckEntries = "Please check the entries."
 contactEditMessage = "Please enter a message."
 contactEditSubject = "Please enter a subject."
-contactAcceptFormPrivacyPolicy = "Please accept the Privacy:hyphen policy."
+contactAcceptFormPrivacyPolicy = "Please accept the Privacy policy."
```

Comment:
"Please accept the privacy policy."
-> privacy should be lower-case, not "Privacy"